.github/workflows/ci.yml

```diff
@@ -2,6 +2,9 @@ name: CI
 
 on: [push, pull_request]
 
+env:
+  SONAR_TOKEN: ${{ secrets.SONARCLOUD_TOKEN }}
```

Comment:
Is it required to use `SONAR_TOKEN` environment variable name as the `SONARCLOUD_TOKEN` has been already taken by the secret? If so, we can rename the secret (see the next comment).